app/src/main/kotlin/edu/stanford/bdh/engagehf/bluetooth/data/mapper/MeasurementToRecordMapper.kt

```diff
@@ -0,0 +1,84 @@
+package edu.stanford.bdh.engagehf.bluetooth.data.mapper
+
+import androidx.health.connect.client.records.BloodPressureRecord
+import androidx.health.connect.client.records.HeartRateRecord
+import androidx.health.connect.client.records.Record
+import androidx.health.connect.client.records.WeightRecord
+import androidx.health.connect.client.units.Mass
+import androidx.health.connect.client.units.Pressure
+import edu.stanford.spezi.core.bluetooth.data.model.Measurement
+import java.time.LocalDateTime
+import java.time.ZonedDateTime
+import javax.inject.Inject
+
+interface MeasurementToRecordMapper {
```

Comment:
As proposed above, this functionalities can be merged in the main mapper of the view model

Reply:
logic moved into repository